backend/src/main/java/com/sscatalog/specialistsservicescatalog/dtos/OfferedServiceDto.java

```diff
@@ -2,18 +2,25 @@
 
```

Comment:
Why this file is named '...Dto.java' in contrast with others named either '...Request' or '...Response' ? (the same question applies for the rest 3 '...Dto.java' files ('ServiceDto.java', 'SpecialistDto.java', 'TagDto.java'))

Reply:
Files named like '..Dto.java' are neither a request object nor a response object so I decided to name them just dtos. Do you have a suggestion for better names for these classes?